tutorials/tutorial-hello-nyc.md

```diff
@@ -34,7 +34,9 @@ CREATE DATABASE nyc_data;
 CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
 ```
 
-Now, download the file [`nyc_data.tar.gz`][nyc_data].
+Now, download the file from the below link.
+
+[`nyc_data.tar.gz`][nyc_data].
```

Comment:
Like the change, but to be consistent with the rest of the article and to correct grammatical issue, perhaps update to:

Next, download the file from the below link:

[` nyc_data.tar.gz `][nyc_data]